go/billing/admin.py

```diff
@@ -36,6 +38,29 @@ def load_credits(self, request, queryset):
         return HttpResponseRedirect(redirect_url)
     load_credits.short_description = "Load credits for selected accounts"
 
+    def is_developer(self, obj):
+        vumi_api = vumi_api_for_user(obj.user)
+        account = vumi_api.get_user_account()
+        return account.is_developer
```

Comment:
I'm a little concerned that this introduces a lot of riak look-ups on the admin pages, but I don't have a better solution so just noting this for now.